test/suites/integration/httproute_creation_test.go

```diff
@@ -82,6 +83,25 @@ var _ = Describe("HTTPRoute Creation", func() {
 		})
 	})
 
+	Context("Order #4: Service, HttpRoute", func() {
+		It("Create lattice resources successfully when no Targets available", func() {
+			httpRoute = testFramework.NewHttpRoute(testGateway, service, "Service")
+
+			// Override port
```

Comment:
Do you need to add more comments here to say that you override the port to cause no port match so that it will cause no Targets in the TG? Or use a more explicit way to construct a no targetPort k8sService?

Reply:
I can add more comments. I doubt creating "no targetPort" K8s service will be used in other place so it will be over-kill to create a method to do that.